### DISCUSSION TITLE
Welcome to tradingview-assistant-chrome-extension Discussions!

### DISCUSSION THREAD

**Discussion by akumidv** [akumidv/tradingview-assistant-chrome-extension#99] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by agentollie** [comment]
Hey there! Is there a way to correct a template if the optimizer is getting the fields incorrect? (and missing some too)

**Comment by JustEFZ** [comment]
Hi Andrei, this assistant looks like exactly what I have needed. Extension installed without issues, but when I add a strategy and then select the "Get Template" button - nothing happens (In fact, I get no response from any button). Am I missing some chrome setting or permission?

**Reply by JustEFZ** [reply]
Please ignore... I needed to restart chrome and restart TV/Supercharts... The Assistant is working well! Everything I needed (for now!)

**Comment by greeenos** [comment]
can you please add symbols from list backtesting them all?

**Comment by Hoadon123** [comment]
I am using your tradingview assistant, I have a question to discuss with you.
Could you write a program where when testing is needed, the strategy will automatically run through each stock ticker. For example, when all 1 code has been checked, the program will automatically save the results and move on to the next code in the list without having to interact. If so, can we discuss this program in more detail, please? Thank you.

**Comment by Hoadon123** [comment]
Hi, I want to discuss with you about a program which you could develop an automatic test strategy base on yours original one, i suggest if you could write that program for me. In this section, i would like to get an automatic which is it will go through each stock in my stock list, with the auto save the result and change to the next stock in the list. If you could write it, please connect me since i want to discuss with you more about that program since i really want to develop it soon. I am looking forward for your replies, connect with me soon. Thank you, Sincerely!

**Comment by akumidv** [comment]
Testing different symbols in whitelist will be implemented in new major version

**Comment by rulphru** [comment]
Hi! Where can I find file with saved template parameters to upload it after assistant update?
